hphp/runtime/base/runtime-option.h

```diff
@@ -606,6 +606,12 @@ struct RuntimeOption {
   F(int64_t, PerfWarningSampleRate, 1)                                  \
   /* Profiling flags */                                                 \
   F(bool, EnableReverseDataMap, false)                                  \
+  /* PPC64 Option: Minimum immediate size to use TOC */                 \
+  F(uint16_t, PPC64minTOCImmSize, 64)                                   \
+  /* PPC64 Option: Dump the number of elements stored in TOC */         \
+  F(bool, PPC64dumpTOCNelements, false)                                 \
+  /* PPC64 Option: Allow lwz and not only ld to be used for TOC read */ \
+  F(bool, PPC64useTOCLwz, false)                                        \
```

Comment:
I think you are right. PPC64useTOCLwz should not be configurable:

Here is what it stands for: if it is set, for immediate values lower than 0xFFFFFFFF (32-bits), a lwz is emitted in spite of a ld. Which means: only 4 bytes of data-block will be used to load an immediate once lwz loads 4 bytes from memory. If PPC64useTOCLwz is not set, 8 bytes will be used (e.g. 0x00000000FFFFFFFF) and a ld will be emmited (load 64 bits from memory).

We were tweaking this parameter to make sure that no misalignment issue would happen by using lwz. And for now, this is not happening.

I will remove PPC64useTOCLwz.